spec/components/playbacktimelabel.spec.ts

```diff
@@ -61,10 +62,14 @@ describe('PlaybackTimeLabel', () => {
       jest.spyOn(playerMock, 'getCurrentTime').mockReturnValue(30);
 
       playbackTimeLabel.configure(playerMock, uiInstanceManagerMock);
+      playerMock.eventEmitter.fireEvent<PlayerEventBase>(
+        {type: PlayerEvent.Ready, timestamp: Date.now()}
+      );
       expect(playbackTimeLabel.getText()).toEqual('01:10');
     });
 
-    it('displays the total time', () => {
+    it('displays the total time mm:ss', () => {
```

Comment:
Could we add the use-case into the test name for easier understanding?

```suggestion
    it('displays the total time mm:ss if duration is lower than 1 hour', () => {
```